README.md

```diff
@@ -110,7 +111,7 @@ def mystery6(s)
 end
 ```
 
-- What is mystery6("goodnight moon")?
-- What is mystery6("Ada Developers Academy")?
-- What is mystery6("Hi, there!")?
+- What is mystery6("goodnight moon")?"moon goodnight"
+- What is mystery6("Ada Developers Academy")?"Developers Academy Ada"
+- What is mystery6("Hi, there!")?"there! Hi,"
```

Comment:
Oh - one tiny thing: you're missing leading spaces on each of these answers. It's a minor thing, but wanted to note it. 

e.g. mystery6("goodnight moon") should be " goodnight moon"